### PR TITLE
Add pylon whitelist option

### PR DESCRIPTION
**Pull request by Brunius**
**When merged this pull request will:**
- Enable missionmakers to set a pylon "whitelist"

### CODE REVIEW

**Comment by PabstMirror** [comment]
Looks pretty good, just replace the tabs with 4 spaces

**Comment by bux** [comment]
And documentation would be nice

**Comment by Brunius** [comment]
Where should said documentation go? I feel a bit weird dumping it in the header here, given it's not really related directly to this function. Should I just put it in the README.md for pylons?

**Comment by mharis001** [comment]
Maybe also use QGVAR macro

**Comment by jonpas** [comment]
Documebtatio is into: `docs/wiki/framework/pylons-framework.html`

**Comment by Brunius** [comment]
@jonpas I can't find that file.
@mharis001 I have just changed it to use the QGVAR macro

**Comment by jonpas** [comment]
Didn't say there was one. ^^
It has to be created, there is nothing about pylons framework like this yet.

**Comment by Brunius** [comment]
@jonpas Most files in that folder are markdown. Should this new file match that format, or break with it?

**Comment by Brunius** [comment]
I've written up a preliminary doc in pylons-framework.md . Let me know if the examples are inappropriate, or if it's not been styled correctly.

**Comment by 654wak654** [comment]
> Looks pretty good, just replace the tabs with 4 spaces
> Ping @654wak654

lgtm

File: docs/wiki/framework/pylons-framework.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: wiki
+title: Pylon Framework
```
Comment (jonpas) [inline docs/wiki/framework/pylons-framework.md]: `Pylons`

File: docs/wiki/framework/pylons-framework.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+description: Explains how to set-up pylon whitelist/blacklist
```
Comment (jonpas) [inline docs/wiki/framework/pylons-framework.md]: `... pylons whitelist and blacklist.`

**Pull request merged**